jenkins/clean_project.py

```diff
@@ -22,68 +24,70 @@
 import sys
 
 
-def Instances(project, age, filt):
-  zones = collections.defaultdict(list)
-  for inst in subprocess.check_output([
-      'gcloud', 'compute', '-q',
-      'instances', 'list',
-      '--format=value(name,zone,creationTimestamp)',
-      '--filter=%s' % filt,
-      '--project=%s' % project]).split('\n'):
-    inst = inst.strip()
-    if not inst:
-      continue
-    name, zone, created_str = re.split(r'\s+', inst)
-    tfmt = 'YYYY-mm-ddTHH:MM:SS'
-    created = datetime.datetime.strptime(created_str[:len(tfmt)], '%Y-%m-%dT%H:%M:%S')
-    if created < age:
-      zones[zone].append(name)
-  return zones
-
+def list_instances(project, age, filt):
+    """List instances."""
```

Comment:
I think bowei hates it more than I do and already promptly ignored all the `missing-doc-string` :-)